uavcan/node/431.GetPorts.1.0.uavcan

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
This definition is brand new and untested. It should be in v0.1 initially. This should be our top priority to get stabilized.